### DISCUSSION TITLE
How do you set-up email when Digital Ocean blocks SMTP?

### DISCUSSION THREAD

**Discussion by alekcunn** [mozilla/hubs#2886] [General Questions & Discussion] [answered]
How do you set-up email when Digital Ocean blocks SMTP?

**Comment by alekcunn** [comment, marked as answer]
I ended up using sendgrid. Port 2525 is unblocked. You can use that.

**Comment by robinkwilson** [comment]
I've had SMTP work for me on my DO deploy using port 587. It was the same day that I deployed.
If you're using sendgrid, use `domain.com` NOT `mail.domain.com` (the default setting when you set it up) and ensure you **verify** the domain with sendgrid.

Did any other configuration for SendGrid work for you? @alekblank

**Reply by alekcunn** [reply]
I don't think I tried 587, but 2525 worked for me. I didn't see 2525 on the sendgrid setup guide. They must've left it out.
